### PR TITLE
Changes for iOS public beta

### PR DESCRIPTION
**Pull request by kylerush**
- fix background image
- delete /temp folder when site is built
- switch back to sign up button instead of sign up form

### CODE REVIEW

**Comment by kylerush** [comment]
@ghigo can you take a look at this dif and make sure it looks ok to you?

File: dist/index.html [hunk 1]
```diff
@@ -332,6 +324,12 @@
               }
               e.preventDefault();
             });
+            //add url parameter to the sign up button
+            var urlParams = document.location.search;
+            urlParams = urlParams.substr(1, urlParams.length);
+            var fullUrl = $('.sign-up-btn').attr('href') + '&' + urlParams;
+            $('.signup-btn').attr('href', fullUrl);
```
Comment (ghigo) [inline dist/index.html]: You should update also this class with .sign-up-btn

File: src/templates/layouts/default.hbs [hunk 2]
```diff
@@ -332,6 +334,12 @@
               }
               e.preventDefault();
             });
+            //add url parameter to the sign up button
+            var urlParams = document.location.search;
+            urlParams = urlParams.substr(1, urlParams.length);
+            var fullUrl = $('.sign-up-btn').attr('href') + '&' + urlParams;
+            $('.signup-btn').attr('href', fullUrl);
```
Comment (ghigo) [inline src/templates/layouts/default.hbs]: same here: .sign-up-btn

**Comment by ghigo** [comment]
Few inline comments

**Comment by ghigo** [comment]
Unable to fully test it locally but the code. I assume you checked that the page works so it LGTM

**Pull request merged**